publication-model/src/main/java/no/unit/nva/model/instancetypes/degree/ArtisticDegreePhd.java

```diff
@@ -0,0 +1,20 @@
+package no.unit.nva.model.instancetypes.degree;
+
+import static no.unit.nva.model.instancetypes.PublicationInstance.Constants.PAGES_FIELD;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.util.Set;
+import no.unit.nva.model.PublicationDate;
+import no.unit.nva.model.pages.MonographPages;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class ArtisticDegreePhd extends DegreePhd {
```

Comment:
I think in case it will be a mistake, migration will be easy to carry out by simply adding @JsonType annotation to DegreePhd.